cfme/tests/services/test_provision_stack.py

```diff
@@ -152,6 +156,18 @@ def stack_created(appliance, provider, order_stack, stack_data):
     yield stack
 
 
+@pytest.fixture
+def stack_not_created_error(appliance, provider, order_stack, stack_data, request):
+    if "test_error_message_azure" in request.node.name:
+        msg = "Orchestration stack deployment error: The supplied password must be"
```

Comment:
I think part of the message is fine. I didn't want to copy-paste the whole one as there's a possibility the exact message will be changed and the test fails